packages/radio-button-plugin/src/packages/RadioGroup.vue

```diff
@@ -33,6 +40,14 @@ export default {
     hasItem() {
       return this.question.choices.length > 0;
     },
+    convertChoices(choices) {
+      return choices.map((item) => {
+        if (typeof item === "string") {
+          return { value: item, disabled: false };
```

Comment:
IMHO, creating the attribute `disabled: false` is redundant -> `{ value: item }` is enough